doc/manual/packages/s3-substituter.xml

```diff
@@ -51,6 +51,16 @@ the S3 URL:</para>
     addressing.</para></note>
   </listitem>
   </varlistentry>
+
+  <varlistentry><term><literal>scheme</literal></term>
+  <listitem>
+    <para>
+      This enables using for http for S3 request for debugging or
+      implementations that don't have https configured.  Should not
+      be used if the cache might contain sensitive information.
```

Comment:
Let's put `<example>https</example> (default) or <example>http</example>` at the start, and then something like:

> This option allows you to disable HTTPS for binary caches which don't support it. HTTPS should be used if the cache might contain sensitive information.